### PR TITLE
Closes #1 replace admiraltemplate placeholders

### LINKED ISSUE #1: Replace admiraltemplate placeholder text/images/code
### Background Information

This repo has been generated from the admiraltemplate repo. Various placeholder files/images need to be updated with metabolic-specific counterparts.

### Definition of Done

Update selected items in the followingg:
- [x] .Rbuildignore
- [x] .lycheeignore
- [x] DESCRIPTION
- [x] README.md
- [x] inst/WORDLIST
- [x] man/figures/logo.png
- [x] vignettes/admiraltemplate.Rmd
- [x] __pkgdown.yml
- [x] tests/testthat.R

Rename:
- [x] R/admiraltemplate-package.R
- [x] project file

### PR DESCRIPTION
**Pull request by manciniedoardo**
Closes #1 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ navbar:
   components:
     getstarted:
       text: Get Started
-      href: articles/admiraltemplate.html
+      href: articles/admiralmetabolic.html
     reference:
       text: Reference
       href: reference/index.html
```
Comment (bundfussr) [inline _pkgdown.yml:67]: I think either you should remove the history icon or add the website-versions article.

File: DESCRIPTION [hunk 2]
```diff
@@ -12,15 +12,16 @@ Description: A toolbox for programming Clinical Data Standards Interchange
...
 URL: https://pharmaverse.github.io/admiraltemplate,
     https://github.com/pharmaverse/admiraltemplate
 Depends: 
     R (>= 4.1)
 Imports:
-    admiral (>= 1.0.0),
-    admiraldev (>= 1.0.0),
+    admiral (>= 1.1.0),
+    admiraldev (>= 1.1.0),
     dplyr (>= 1.0.5),
```
[the remark this answers is not in the document]
Comment (manciniedoardo) [reply DESCRIPTION:25]: i think better to include (other ext packages do too)

File: _pkgdown.yml (hunk 1 above)
Comment (manciniedoardo) [reply _pkgdown.yml:67]: done

File: DESCRIPTION (hunk 2 above)
[the remark this answers is not in the document]
Comment (bundfussr) [reply DESCRIPTION:25]: In admiralonco we only have:
```
Imports:
    admiral (>= 1.0.0),
    admiraldev (>= 1.0.0),
    dplyr (>= 0.8.4),
    lifecycle (>= 0.1.0),
    lubridate (>= 1.7.4),
    magrittr (>= 1.5),
    rlang (>= 0.4.4),
    tidyselect (>= 1.0.0)
```

File: DESCRIPTION [hunk 3]
```diff
@@ -1,9 +1,9 @@
 Type: Package
-Package: `<<INSERT PACKAGE EXT NAME>>'
-Title: ADaM in R Asset Library - Extension Package for `<<INSERT TA>>`
+Package: admiralmetabolic
+Title: Metabolism Extension Package for ADaM in 'R' Asset Library
 Version: 0.1.0
```
[the remark this answers is not in the document]
Comment (manciniedoardo) [reply DESCRIPTION]: true, changed to 0.0.0

File: DESCRIPTION (hunk 2 above)
[the remark this answers is not in the document]
Comment (manciniedoardo) [reply DESCRIPTION]: ok - will use same thanks

File: _pkgdown.yml [hunk 4]
```diff
@@ -1,14 +1,14 @@
-url: https://pharmaverse.github.io/admiraltemplate
+url: https://pharmaverse.github.io/admiralmetabolic
 
-template:
+metabolic:
```
Comment (rossfarrugia) [inline _pkgdown.yml:3]: i think this was an accidental find/replace - it should be reverted

**Review by rossfarrugia — CHANGES_REQUESTED** [review]

File: _pkgdown.yml (hunk 4 above)
Comment (manciniedoardo) [reply _pkgdown.yml:3]:
```suggestion
template:
```

**Review by rossfarrugia — APPROVED** [review]
